spec/lib/generators/forem/dummy/templates/config/initializers/simple_form.rb

```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
My concern would be that if `simple_form` changed their defaults, then these "defaults" that we have here would be out of date, and therefore incorrect.